src/pages/style.css

```diff
@@ -129,6 +129,26 @@ Mobile & tablet visibility helpers, specific to dev-dot.
   }
 }
 
+/*
+***
+* Screen reader only global helper
+***
+* Visually hides content so it is only available to screen readers. Sourced
+* Scott O'Hara's Inclusively Hidden article.
+*
+* https://www.scottohara.me/blog/2017/04/14/inclusively-hidden.html
+***
+*/
+.g-screen-reader-only:not(:focus):not(:active) {
```

Comment:
Non-blocking question: going forward do you think we should aim to use this `className` rather than `VisuallyHidden`?

Reply:
Yep!